src/EnumSet.php

```diff
@@ -54,19 +60,17 @@ public function __construct($enumeration)
                 __NAMESPACE__ . '\Enum'
             ));
         }
+        if (version_compare(phpversion(), '5.6.3', '<')) {
+            $this->integerSize = 4*8;
+        }else {
+            $this->integerSize = PHP_INT_SIZE*8;
+        }
```

Comment:
`$this->integerSize = PHP_INT_SIZE * 8;` should work on all versions. The issue before was calculating the value of `integerSize` on compile time which wasn't supported before 5.6.